android/app/src/main/java/com/androidstartupperfapp/MainActivity.java

```diff
@@ -43,6 +43,8 @@ private void maybeGoBackToLandingScreen() {
 
       // Show the Android landing screen fragment to simulate back button press navigation.
       landingScreenFragment.showAsync(this);
+      
+      // More state to deal with so that the React app knows when not to navigate?
     }
   }
```

Comment:
[4/4]

This is a question we have to answer if we were to follow the docs. Right now, I have no answer to this without overengineering the solution with more bridges or state.